src/app/components/new-activity-modal/new-activity-modal.component.scss

```
@@ -0,0 +1,38 @@
+@import "/src/styles/colors";
+
+.container-new {
+  overflow-y: scroll;
+
+  .close {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .header-title {
+    text-align: center;
+    color: $color-titles;
+  }
+
+  .container-body-new {
+    margin: 0 50px 0 50px;
```

```suggestion
    margin: 0 50px;
```
This should do the trick.